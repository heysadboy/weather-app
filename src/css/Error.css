#error-container {
    padding-top: 50px;
}