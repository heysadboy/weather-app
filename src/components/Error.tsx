import '../css/Error.css';


const Error = () => {
    return (
        <div id="error-container" className="ui center aligned container">
            <div className="ui negative message">
                <div className="header">We're sorry some error occurred</div>
                <p>Please try again later</p>
            </div>
        </div>
    );
}

export default Error;