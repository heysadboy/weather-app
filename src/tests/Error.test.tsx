import renderer from 'react-test-renderer';
import Error from '../components/Error';


test('Test Error', () => {
    const component = renderer.create(
        <Error />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});