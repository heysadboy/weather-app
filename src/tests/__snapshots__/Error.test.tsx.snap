// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Error 1`] = `
<div
  className="ui center aligned container"
  id="error-container"
>
  <div
    className="ui negative message"
  >
    <div
      className="header"
    >
      We're sorry some error occurred
    </div>
    <p>
      Please try again later
    </p>
  </div>
</div>
`;
